### PR TITLE
Update maintainer-list.nix

### PR DESCRIPTION
**Pull request by thefenriswolf**
add thefenriswolf, maintainer of tensorman

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by prusnak** [comment]
Maybe just merge this commit into https://github.com/NixOS/nixpkgs/pull/73452 and close this PR.

**Comment by thefenriswolf** [comment]
> 
> 
> Maybe just merge this commit (as a separate commit!) into #73452 and close this PR.

I don't get what I am supposed to do! Should this be a separate PR or not?
Please just ELI5 I'm lost

**Comment by prusnak** [comment]
Not a separate PR, but put a separate commit into #73452 (so this will contain two commits: one new package, one new maintainer)
